pkg/networks/commands.go

```diff
@@ -22,6 +24,36 @@ func (config *NetworksConfig) Check(name string) error {
 	return fmt.Errorf("network %q is not defined", name)
 }
 
+func (config *NetworksConfig) CheckDaemon(daemon string) (bool, error) {
+	f := func(p string) (bool, error) {
```

Comment:
Is there any reason not to make this a "named" local function instead? It doesn't seem to be a closure, so I don't know why it is defined as a function pointer inside the method.

Reply:
Fixed